tests/unit/moduleapi/testrdb.tcl

```diff
@@ -128,5 +128,161 @@ tags "modules" {
                 }
             }
         }
+
+        test {Diskless load swapdb, module can use RedisModuleEvent_ReplAsyncLoad CREATE and COMPLETED events on async_loading} {
+            start_server [list overrides [list loadmodule "$testmodule 2"]] {
```

Comment:
i see you added `external:skip`, but maybe we should also add the `repl` tag like other tests (they kinda cover the same thing, but who knows what will someone try to skip...)

Reply:
These 2 new tests are under `tags {repl} {`, isn't already tagged with that?